### PR TITLE
ISV Riggs

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request

Adds a new ship, the Riggs-class Sloop, based on the Rebel Rigger from FTL. 
rigger

Features include:
riggercargo
a mech bay, conveniently located at the front of the ship,

riggercentral
 a robotics workshop with a posibrain,

riggermedsec
and a dedicated (if unstocked) armory space.

Notable shortcomings include a poorly-stocked medbay and complete lack of cargo facilities outside of the ore redemption machine. Better get salvaging!

## Why It's Good For The Game

we test ships here

## Changelog
:cl:
add: Added a new ship.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
looks really good, cant wait to test this ship!

**Comment by Apogee-dev** [comment]
goofed on the map format, whoops

**Review by triplezeta — CHANGES_REQUESTED** [review]
image


image

File: _maps/shuttles/shiptest/rigger.dmm [hunk 1]
```diff
@@ -0,0 +1,5148 @@
...
+/area/ship/maintenance/port)
+"jq" = (
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/structure/sign/poster/contraband/random{
+	pixel_x = -32
+	},
+/obj/machinery/navbeacon/wayfinding/atmos{
+	name = "navigation beacon
```
Comment (MarkSuckerberg) [inline _maps/shuttles/shiptest/rigger.dmm]:
```suggestion
	name = "navigation beacon"
```

File: _maps/shuttles/shiptest/rigger.dmm [hunk 2]
```diff
@@ -0,0 +1,5150 @@
...
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/turf/open/floor/plasteel/dark,
+/area/ship/bridge)
+"mn" = (
+/obj/structure/sign/warning/nosmoking{
+	pixel_y = -32
+	},
+/obj/machinery/navbeacon{
+	location = "med"
```
Comment (MarkSuckerberg) [inline _maps/shuttles/shiptest/rigger.dmm:862]:
```suggestion
	codes_txt = "patrol;next_patrol=canteen";
	location = "med"
```

**Pull request merged**